x-pack/plugins/synthetics/server/routes/monitor_cruds/add_monitor_project.ts

```diff
@@ -74,6 +74,10 @@ export const addSyntheticsProjectMonitorRoute: SyntheticsRestApiRouteFactory = (
       };
     } catch (error) {
       server.logger.error(`Error adding monitors to project ${decodedProjectName}`);
```

Comment:
```suggestion
      server.logger.error(`Error adding monitors to project ${decodedProjectName}, kibana space does not exist.`);
```

Reply:
I think this should stay as it is because at this point the error could be anything, not necessarily that we didn't find the space.

Reply:
got it.